### DISCUSSION TITLE
1.17 support

### DISCUSSION THREAD

**Discussion by kaelhufford** [minecraft-dev/MinecraftDev#959] [Q&A] [answered]
Hello, does anyone know when this will start to support spigot plugin development for 1.17 part one? 
Thanks!

**Comment by RedNesto** [comment, marked as answer]
Hi, 1.17 is not even released yet, why ask so early ?

As always the plugin doesn't usually have anything to do to support newer versions of spigot.

We'll update the project creator available versions after the frameworks that require manual updates release their first versions for 1.17, which we'll do when we have enough time.

**Reply by kaelhufford** [reply]
Okay, thanks!

**Comment by DenWav** [comment]
Forge and Fabric (and quilt) and Paper (and probably others) are all making really big changes so there's definitely going to be things which may be broken at least for a little bit.
